### DISCUSSION TITLE
Lolin d1 mini not working

### DISCUSSION THREAD

**Discussion by lazyjalokin** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#783] [General]
I used a lolin board with a red llc. I have tried everything. I'm beginning to suspect, that the lolin board it self is the problem. I have tried several iterations of the completed circuit.
The webserver is working as it should, but I can not get it to work with the spa (100101). I have tried every combination of pinout and pump models. The lolin board should match the wemos board, but am I missing something? 
With differentiere setting I can get the power button to make a slight sound, and at one point, there was green light in the power button, but still no response. Any ideas from anyone? Help is appreciated.

**Comment by visualapproach** [comment]
Hi there! I guess there may be connection issues. Some new models seem to be picky about the signals. But I recommend posting pictures of your device so we can complain about the soldering ;)
And also perform a few tests (links in links.html):
...inputs/ 
...debug-on/
...hwtestinfo.html

Also state the firmware version 
/VA

**Reply by lazyjalokin** [reply]
Hi again. I don't know where the "links.html" can be found. Sorry if it is obvious, but I have searched everywhere.. Thx in advance.

**Reply by visualapproach** [reply]
It's a file on the ESP. You can see all files from the menu/directory

**Comment by lazyjalokin** [comment]
Hi!
Thanks for your quick answer and for a great project.

Firmware: 2024-06-17-1334
Model: MIAMI2021
RSSI: -60

I have never ordered a pcb before, so I made it on some "perfboard?"  I had lying around.

![20240806_155906](https://github.com/user-attachments/assets/e9e10858-dff2-4f92-a2c9-8d74cc099f88)
![20240806_155856](https://github.com/user-attachments/assets/96df9aa9-4dbc-4cbb-a37a-700c8a82443b)

Will try the links you sent and report back.

**Reply by visualapproach** [reply]
Thx, update to latest version for better chances
